extensions/typescript-language-features/package.json

```diff
@@ -73,6 +73,9 @@
     "onLanguage:jsonc",
     "onWalkthrough:nodejsWelcome"
   ],
+  "extensionDependencies": [
+		"ms-vscode.remote-repositories"
```

Comment:
@joyceerhl I had to add this to get JS/TS to see the `ms-vscode.remote-repositories` extension. However we can't ship with this dependency

Is there a way to mark this dependency as optional or some other way to ensure the JS/TS extension is running in the same extension host as `ms-vscode.remote-repositories` on web?

Reply:
The best way to do this is to update the extension host affinity in vscode.dev.